code/game/machinery/computer/orders/order_items/mining/order_consumables.dm

```diff
@@ -13,6 +13,11 @@
 	item_path = /obj/item/storage/medkit/brute
 	cost_per_order = 600
 
+/datum/orderable_item/consumables/medkit_fire
+	item_path = /obj/item/storage/medkit/fire
+	desc = "For emergency magmatic burn relief."
+	cost_per_order = 700
```

Comment:
shouldnt this be the same price as brute?

Reply:
I increased the price compared to the brute as sources of burn damage on lavaland: Drake Fire, Lava, etc can lead to burn wounds which can lead to infections etc, while currently, nothing on lavaland can cause you bleeding damage, or broken bones.

Reply:
though burn kit doesn't have mesh to fix said infections maybe at max hercuri which is a nightmare in its own